src/NSubstitute/Proxies/CastleDynamicProxy/CastleDynamicProxyFactory.cs

```diff
@@ -35,6 +36,11 @@ public object GenerateProxy(ICallRouter callRouter, Type typeToProxy, Type[] add
             return proxy;
         }
 
+        public TypeBuilder DefineDynamicType(string typeName, TypeAttributes flags)
```

Comment:
Could you please add a quick `///` xmldoc comment to this to explain what it does/is used for?
We haven't normally done this for this project, but I'd like to start, especially when it is not necessarily immediately obvious (I think `GenerateProxy()` is more self-explanatory so a comment is less useful there).

Reply:
Added 👍